File: Java/src/gestorAplicacion/Padre/Mensaje.java
package gestorAplicacion.Padre;

import java.io.Serializable;

public abstract class Mensaje implements Serializable {
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	String titulo;
	String cuerpo;
		
	public Mensaje(String titulo, String cuerpo) {
		super();
		this.titulo = titulo;
		this.cuerpo = cuerpo;
	}
	
	@Override
	public abstract String toString();

	protected String getTitulo() {
		return titulo;
	}

	protected void setTitulo(String titulo) {
		this.titulo = titulo;
	}

	protected String getCuerpo() {
		return cuerpo;
	}

	protected void setCuerpo(String cuerpo) {
		this.cuerpo = cuerpo;
	}
	
}

File: Python/gestorAplicacion/Padre/Mensaje.py

#Serializable 
class Mensaje:

    #_SERIALVERSIONUID = 1

    def __init__(self, titulo, cuerpo):
        self.titulo = titulo
        self.cuerpo = cuerpo

    def getTitulo(self):
        return self.titulo

    def setTitulo(self, titulo):
        self.titulo = titulo

    def getCuerpo(self):
        return self.cuerpo

    def setCuerpo(self, cuerpo):
        self.cuerpo = cuerpo
